subsys/bluetooth/controller/Kconfig

```diff
@@ -371,6 +371,17 @@ config BT_CTLR_PRIVACY
 	  Enable support for Bluetooth v4.2 LE Controller-based Privacy feature
 	  in the Controller.
 
+config BT_CTLR_WL_SIZE
+	int "LE Controller-based Privacy White List size"
+	depends on BT_CTLR_FILTER
+	default 8
+	range 1 8  if  (SOC_COMPATIBLE_NRF || SOC_OPENISA_RV32M1_RISCV32)
+	range 1 16 if !(SOC_COMPATIBLE_NRF || SOC_OPENISA_RV32M1_RISCV32)
+	help
+	  Set the size of the White List for LE Controller-based Privacy.
+	  On nRF5x-based controllers, the hardware imposes a limit of 8 devices.
+	  On OpenISA-based controllers, the hardware imposes a limit of 8 devices.
```

Comment:
@George-Stefan is this true?

Reply:
The 8 entries limit is there because of time constraints.
Let's keep it like this.
@alexandru-porosanu-nxp  please confirm